tests/test_helpers.py

```diff
@@ -4,59 +4,59 @@
 from vitrina.helpers import get_selected_value, get_filter_url
 
 
+class MockFacetForm:
+    selected_facets = []
+
+    def __init__(self, selected_facets):
+        self.selected_facets = selected_facets
+
+
 @pytest.mark.django_db
-def test_get_selected_value(rf: RequestFactory):
-    request = rf.get('/', {'key': 'value'})
-    selected_value = get_selected_value(request, 'key', is_int=False)
+def test_get_selected_value():
+    facet_form = MockFacetForm(['key_exact:value'])
```

Comment:
```suggestion
    facet_form = Mock(selected_facets=['key_exact:value'])
```

Simply use the [Mock](https://docs.python.org/3/library/unittest.mock.html#the-mock-class) class. Please do the same thing in other places and remove `MockFacetForm` class.